src/sentry/static/sentry/app/components/contextData.jsx

```diff
@@ -125,7 +123,7 @@ class ContextData extends React.Component {
           <span
             key="value"
             className={
-              (valueInfo.isString ? 'val-string' : 'val-repr') +
+              (valueInfo.isString ? 'val-string' : '') +
```

Comment:
do we no longer need 'val-repr'? I'm not too familiar with contextData, but if it's not needed we could remove it from the less file as it seems to not be used anywhere else.

Reply:
Gladly removing